sagemathinc/cocalc
# Landing page tweaks

# Description
* stying of landing pages, some reorgs, etc.
* dispatch topic specific actions for some "start now" buttons

# Testing Steps
- besides the looks, the main tedious task of testing was to
  * sign out of the app
  * click on a "start now" button (terminal, jupyter, R, x11, ...)
  * check what happens
  * rinse/repeat for another button type :repeat: 
- deploying it in "test"

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
